back-end/server/db/models/activity.js

```diff
@@ -14,6 +14,10 @@ const Activity = db.define('activity', {
   imageUrl: {
     type: Sequelize.STRING,
     allowNull: false
+  },
+  points: {
+    type: Sequelize.INTEGER,
+    defaultValue: 0
```

Comment:
Me and Andrew were talking about activity dashboard for the user, and we realized we might have to do some front-end math to show points for each activity. But, it makes more sense to store it in the backend with models to avoid the front end math at multiple places.

Reply:
In order to have the points for each activity displayed in each location related to the activity, every product and category related to every activity will need to be eager loaded every time an activity or activity list is shown. It is just easier to store that point value on the activity itself. We thought the tradeoff in space complexity was worth the ease of use in this instance.